neqo-transport/src/frame.rs

```diff
@@ -495,18 +495,18 @@ pub fn decode_frame(dec: &mut Decoder) -> Res<Frame> {
         }
         FRAME_TYPE_STREAM...FRAME_TYPE_STREAM_MAX => {
             let s = dv!(dec);
-            let o = if t & STREAM_FRAME_BIT_OFF != 0 {
-                dv!(dec)
-            } else {
+            let o = if t & STREAM_FRAME_BIT_OFF == 0 {
```

Comment:
This one is a little annoying.  I think of non-zero testing the other way around.  Oh well.

Reply:
I can disable it if you would like, since it is more explicit for our use case.

Reply:
Nah.  Better to minimize the warnings, even if they sometimes go against preconceptions.  Warnings reduce variation in coding styles, which is better than pandering to those preconceptions.

Reply:
:ok_hand: